src/reaper_osara.cpp

```diff
@@ -3956,6 +3956,37 @@ void cmdManageTempoTimeSigMarkers(Command* command) {
 	Main_OnCommand(40256, 0); // Tempo envelope: Insert tempo/time signature change marker at edit cursor...
 }
 
+void cmdSelectItemsUnderEditCursorOnSelectedTracks(Command* command) {
+	vector<MediaItem*> items;
+	for (int i = 0; i < CountTracks(nullptr); ++i) {
+		MediaTrack* track = GetTrack(nullptr, i);
+		for (int j = 0; j < CountTrackMediaItems(track); ++j) {
+			MediaItem* item = GetTrackMediaItem(track, j);
+			items.push_back(item);
+		}
+	}
+	PreventUIRefresh(1); // SWS use this
+	Main_OnCommand(40289,0); // unselect all items
+	int i;
+	double cursorPosition=GetCursorPosition();
+	for (i=0;i<(int)items.size();++i) {
```

Comment:
Because you have a vector of MediaItems here and you don't need the index, you can just do:
`for (MediaItem* item: items) {`